### PR TITLE
DIG-14 Correct SUMit reminder date format and add test

### PR DESCRIPTION
**Pull request by rjacobs31**
Fixes #257.

### CODE REVIEW

**Comment by rjacobs31** [comment]
Hey, @msinovcic. Would you please take a look at the fix?

File: content/tests.py [hunk 1]
```diff
@@ -464,6 +465,21 @@ def test_end_of_event_task_event_with_answers_for_multiple_learners_first_awarde
 
         self.assertEquals(awarded_badge.count(), 0)
 
+    @patch("mobileu.tasks.mail_managers")
+    def test_sumit_counts(self, mocked_mail_managers):
+        s = SUMit.objects.create(name='Blarg',
+                                 course=self.course,
+                                 activation_date=datetime.now()+timedelta(hours=12),
+                                 deactivation_date=datetime.now()+timedelta(hours=24))
+        s.save()
```
Comment (msinovcic) [inline content/tests.py]: This is good, just thinking maybe you should also check that the counts returned by get_question_counts method are correct.

File: content/tests.py [hunk 2]
```diff
@@ -464,6 +465,96 @@ def test_end_of_event_task_event_with_answers_for_multiple_learners_first_awarde
 
         self.assertEquals(awarded_badge.count(), 0)
 
+    def test_sumit_counts(self):
+        s = SUMit.objects.create(name='Blarg',
+                                 course=self.course,
+                                 activation_date=datetime.now()+timedelta(hours=12),
+                                 deactivation_date=datetime.now()+timedelta(hours=24))
+        s.save()
```
Comment (msinovcic) [inline content/tests.py]: there is no need for save. create creates and saves the object.

**Comment by msinovcic** [comment]
remove the save and you good to go 👍

**Pull request merged**